### ISSUE TITLE
Add ability to hide cmd window.

### ISSUE THREAD

**Issue by abrelsfo** [sindresorhus/opn#27]
On windows when you run opn it opens a new .cmd every time you call it. It would be really nice if there was an option to hide those.

**Comment by sindresorhus** [comment]
Happy to receive a pull request, but I personally have no idea whether that's possible or how.

**Comment by ddantzler** [comment]
> ...I personally have no idea whether that's possible or how.

This is an old request, but I ran into it myself.

**On win32, you are running:**
cmd /c start [app] [args]

**You need to run:**
cmd /c start /B [app] [args]

If you want to make the shell hidden. Curious, why on win32 do you push '""' after start?

**Comment by gotdibbs** [comment]
To answer your question @ddantzler I'm guessing that the "" is intended to the be [`title` parameter](https://technet.microsoft.com/en-us/library/cc770297(v=ws.11).aspx).

I tried looking into this just now, as when I try to launch VS Code using this utility a command prompt window is opened every time on Windows.

After attempting to add in the `/b` parameter to the `start` command, I wasn't able to resolve this issue for me. It appears that for most of the cases however, the `/c` parameter currently being used on `cmd` terminates the command prompt windows that results, but not in this specific case (no idea why yet, but likely has something to do with VS Code being an electron app).

In summary, I think the existing code covers most use cases and `/b` doesn't appear to fix my use case so I have not made the update. You can check and see if it will fix yours by going to start -> run and entering `cmd /c start "" /b [command/file]` where `[command/file]` is what you'd be opening with `opn`.

**Comment by ddantzler** [comment]
Interesting, what version of Windows are you on? I am in an environment running Windows 7. Same use case for me, desiring to launch VS Code and/or Atom. The /b flag is working for me, so for now I am using my own fork.

Before implementing, I tested using the exact method you described.

cmd /c start /b notepad

https://technet.microsoft.com/en-us/library/cc770297(v=ws.11).aspx

**Comment by ddantzler** [comment]
I wonder why the combination of cmd and start?

**Comment by ddantzler** [comment]
I mean to say why does opn use both? Additionally in my testing, even running notepad without /b results in a leftover shell, eliminating electron. It’s the combination of cmd with start that makes the /c basically non-effective.

**Comment by gotdibbs** [comment]
@ddantzler Windows 10 v.1607. You can see how I'm testing the command in the gif below.

It appears not to matter however if I use `/b` or not, nor if I omit `start`. Same net result.

![windows10repro](https://user-images.githubusercontent.com/1540648/29435250-d9494a4c-836b-11e7-8c20-a07e346e1ae8.gif)

**Comment by gotdibbs** [comment]
Upon further investigation, the issue appears to be that `code` is actually the alias for `bin\code.cmd` and appears to do some trickery related to the fact that it is intended to be used in the command line. Launching the exe instead (tested using the full path) seems to work fine as does using [VS Code's custom URL scheme](https://code.visualstudio.com/docs/editor/command-line#_opening-vs-code-with-urls).

For my intents and purposes I am going with using the URL handling mechanism so that I may cover Mac and Windows.

**Comment by ddantzler** [comment]
Great find!! 👍 

I’ll switch to the url mechanism too.

Deron

**Comment by kevva** [comment]
The `b` flag was added by default in https://github.com/sindresorhus/opn/commit/eb99a736e8b277f16277765aae5241a67962782c so I think we can consider this fixed.
